ocp_resources/virtual_machine_export.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.persistent_volume_claim import PersistentVolumeClaim
+from ocp_resources.resource import NamespacedResource
+from ocp_resources.virtual_machine import VirtualMachine
+from ocp_resources.virtual_machine_snapshot import VirtualMachineSnapshot
+
+
+class VirtualMachineExport(NamespacedResource):
+    """
+    VirtualMachineExport object.
+    """
+
+    api_group = NamespacedResource.ApiGroup.EXPORT_KUBEVIRT_IO
+
+    class SourceKind:
+        VM = VirtualMachine.kind
+        VM_SNAPSHOT = VirtualMachineSnapshot.kind
+        PVC = PersistentVolumeClaim.kind
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        client=None,
+        teardown=True,
+        token_secret_ref=None,
+        source_api_group=None,
+        source_kind=None,
+        source_name=None,
+        timeout=TIMEOUT_4MINUTES,
+        delete_timeout=TIMEOUT_4MINUTES,
+        yaml_file=None,
+        **kwargs,
+    ):
+        if not token_secret_ref:
```

Comment:
you shouldn't raise on this `token_secret_ref` if the user pass yaml_file.

Reply:
right, changed it, thanks